example-narrative.md

```diff
@@ -45,7 +47,7 @@ Let's add some configuration to reduce the width of the image and add color to t
 
 **Input:**
 ```
-![Image](https://placehold.co/800x100) <!--{ width="300" }-->
+![Image](https://placehold.co/800x100) <!--{ width="800" }-->
```

Comment:
If we change the width here, then we'd also need to change it in line 57. Originally, the intention was to show that an image that is originally 800px wide can be rendered as 300 wide instead of the original size, so I'd be in favor of keeping the example with "300" width

![image](https://github.com/user-attachments/assets/88f0bd40-33d3-40e5-9e78-cc91fd7e6ca8)

Reply:
So I misunderstood the meaning of the widths parameter in the comment. Please ignore my change request concerning this line.